staging/src/k8s.io/apimachinery/pkg/util/wait/error_test.go

```diff
@@ -49,7 +49,7 @@ func TestInterrupted(t *testing.T) {
 		want bool
 	}{
 		{
-			err:  ErrWaitTimeout,
+			err:  ErrorInterrupted(nil),
```

Comment:
don't change this test case util the wait.ErrWaitTimeout is unused by all other packages.

And we should remove this test case once wait.ErrWaitTimeout is removed.